drivers/scheduler/k8s/k8s.go

```diff
@@ -2929,6 +2936,55 @@ func (k *K8s) getExpectedAutopilotSizeForPvc(pvc *v1.PersistentVolumeClaim, apRu
 	return expectedPVCSize, nil
 }
 
+// ValidateEvents verifies proper alerts and events on resize completion
+func (k *K8s) ValidateEvents(pvc *v1.PersistentVolumeClaim, apRules *apapi.AutopilotRuleList) error {
+
+	// wait until autopilot posts all events
+	time.Sleep(time.Minute)
+
+	successEvens := []string{
+		"Initializing => Normal",
+		"Normal => Triggered",
+		"Triggered => ActiveActionsPending",
+		"ActiveActionsPending => ActiveActionsInProgress",
+		"ActiveActionsInProgress => ActiveActionsTaken",
+	}
```

Comment:
we have to add  ActiveActionsTaken => Normal event after cool down period finished